queue/src/main/resources/org/killbill/queue/dao/QueueSqlDao.sql.stg

```diff
@@ -207,6 +207,19 @@ updateOnError(tableName) ::= <<
     ;
 >>
 
+updateCreatingOwner(tableName) ::= <<
+    update <tableName>
+    set
+      creating_owner = :newCreatingOwner
+      , processing_owner = NULL
+      , processing_available_date = :now
+      , processing_state = 'AVAILABLE'
+    where
+      <reapWhereClause()>
+    limit :max
```

Comment:
I believe this won't work on PostgreSQL unfortunately (could be the cause of the CI PostgreSQL failure).